src/VisualStudio/SetupInteractive/VisualStudioSetupInteractive.csproj

```diff
@@ -54,34 +58,28 @@
     <ProjectReference Include="..\CSharp\Repl\CSharpVisualStudioRepl.csproj">
       <Project>{737ff62c-f068-4317-84d0-bfb210c17a4e}</Project>
       <Name>CSharpVisualStudioRepl</Name>
-      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3bPkgDefProjectOutputGroup</IncludeOutputGroupsInVSIX>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
```

Comment:
I know very little about how this works.  In my first attempt, I changed "%3b" to ";" based on a StackOverflow issue and then set the value to BuildProjectOutputGroup;PkgDefProjectOutputGroup in every project reference.  Any file that subsequently produced an error about PkgDefProjectOutputGroup had all occurrences removed.

I'm happy to change it, as long as you tell me what to change. :)